README.md

```diff
@@ -12,29 +12,30 @@ The documents are written in RST and are built using [Sphinx](http://www.sphinx-
 - Every build will generate docs for all versions maintained.
 - The root documentation points to the latest released verion. Navigate to `master` version for the development version of this documentation.
 
+#### Pre-requisites
+
+* Python 3
+* Nodejs
+* NPM
+* pip3 install -U Sphinx
```

Comment:
AFAIK NodeJs is not required. `pip` and  `Make` is required.
Also `pip3 install -U Sphinx` is not required, in fact, what required is `pip3 install -r requirements.txt` before running `make`, `sphinx` will be installed as part of that.

Reply:
Will remove pip3 install -U Sphinx. As far as NodeJS was concerned, I have only retained it. This was there earlier. I have just moved the pre-requisites before building the document.